frontend/src/components/MapView/LeftPanel/AnticipatoryActionPanel/index.tsx

```diff
@@ -0,0 +1,270 @@
+    [AADateSameDate],
+  );
+  const readySev = React.useMemo(
+    () =>
+      AADateSameDate.filter(
+        x => x.category === 'Severo' && x.phase === 'Ready',
+      ),
+    [AADateSameDate],
+  );
+  const setMod = React.useMemo(
+    () =>
+      AADateSameDate.filter(
+        x => x.category === 'Moderado' && x.phase === 'Set',
+      ),
+    [AADateSameDate],
+  );
+  const readyMod = React.useMemo(
+    () =>
+      AADateSameDate.filter(
+        x => x.category === 'Moderado' && x.phase === 'Ready',
+      ),
+    [AADateSameDate],
+  );
+
+  // TODO - LEVE is "MILD" and should be added as a new category, see Figma.
+  const setMild = React.useMemo(
+    () =>
+      AADateSameDate.filter(x => x.category === 'Leve' && x.phase === 'Set'),
+    [AADateSameDate],
+  );
+  const readyMild = React.useMemo(
+    () =>
+      AADateSameDate.filter(x => x.category === 'Leve' && x.phase === 'Ready'),
+    [AADateSameDate],
+  );
+
+  // eslint-disable-next-line no-console
+  console.log(setMild, readyMild);
+
+  // TODO: implement NA and NY
+  // NA means that no proba is above the trigger for that district
+  // NY means that the district is not monitored yet (no rows for the district)
+  const na = React.useMemo(() => [], []);
+  const ny = React.useMemo(() => [], []);
+
+  // -1 means all
+  const selectedWindowIndex = windows.findIndex(x => x === selectedWindow);
+
+  const headerRow: RowProps = {
+    id: 'header',
+    iconContent: null,
+    windows: selectedWindowIndex === -1 ? windows.map(x => []) : [[]],
+    header:
+      selectedWindowIndex === -1
+        ? [...windows]
+        : [windows[selectedWindowIndex]],
+  };
+
+  function getWindowData(data: AnticipatoryActionData[], window: number) {
+    return data
+      .filter(x => x.windows === windows[window])
+      .map(x => ({ name: x.district, isNew: false }));
+  }
+
+  function windowData(data: AnticipatoryActionData[]) {
+    return selectedWindowIndex === -1
+      ? windows.map((x, index) => getWindowData(data, index))
+      : [getWindowData(data, selectedWindowIndex)];
+  }
+
+  const rows: RowProps[] = [
+    headerRow,
+    {
+      id: 'set_sev',
+      iconContent: AAIcons.set_sev,
+      windows: windowData(setSev),
+    },
+    {
+      id: 'ready_sev',
+      iconContent: AAIcons.ready_sev,
+      windows: windowData(readySev),
+    },
+    {
+      id: 'set_mod',
+      iconContent: AAIcons.set_mod,
+      windows: windowData(setMod),
+    },
+    {
+      id: 'ready_mod',
+      iconContent: AAIcons.ready_mod,
+      windows: windowData(readyMod),
+    },
+    {
+      id: 'na',
+      iconContent: AAIcons.na,
+      windows: windowData(na),
+    },
+    {
+      id: 'ny',
+      iconContent: AAIcons.ny,
+      windows: windowData(ny),
+    },
+  ];
+
+  return (
+    <div className={classes.anticipatoryActionPanel}>
+      {/* header */}
+      <div className={classes.headerWrapper}>
+        {/* Title area */}
```

Comment:
It's nice to add comments but some of these feel a bit superfluous.

Reply:
Yes i added them when i originally create the layout. now they don't really serve any purpose, i am removing them